desc/integrals/basis.py

```diff
@@ -227,14 +227,22 @@ def harmonics(self):
         Transform Fourier interpolant harmonics to Nyquist trigonometric
         interpolant harmonics so that the coefficients are all real.
 
+        The order of the returned coefficient array
+        matches the Vandermonde matrix formed by an outer
+        product of Fourier and Chebyshev matrices with order
+        [sin(k𝐱), ..., sin(𝐱), 1, cos(𝐱), ..., cos(k𝐱)]
+        ⊗ [T₀(𝐲), T₁(𝐲), ..., Tₙ(𝐲)]
+
+        When ``self.X`` is even the sin(k𝐱) coefficient is zero and is excluded.
```

Comment:
```suggestion
        When ``self.X`` is even the sin(k𝐱) coefficients are zero and is excluded.
```
right?

Reply:
sure, feel free to press commit before approval